feature/bgp/gracefulrestart/ate_tests/err/README.md

```diff
@@ -0,0 +1,289 @@
+  * use `gNOI.killProcessRequest_Signal_KILL` this time as per `gNOI proto`
+  * Expected behavior is the same as RT-1.35.2.a<br>
+ 
+  ***`b. Restarting DUT speaker whose BGP process was killed abruptly after ERR policy was removed.`***
+  * Expected behavior is the same as RT-1.35.2.b<br><br><br>
+
+   
+
+  
+
+**RT-1.35.4**
+
+  ***`a. DUT Helper for a restarting EBGP speaker whose BGP process was gracefully killed. In this case ERR policy is attached to the BGP neighborship.`***
+  * Send Graceful restart trigger from ATE:Port1
+  * Expected behavior is the same as RT-1.35.2.a
+  * Repeat by sending Graceful restart trigger from ATE:Port2 with the same expected behavior as RT-1.35.2.a<br>
+
+  ***`b. DUT Helper for a restarting EBGP speaker whose BGP process was gracefully killed after ERR policy was removed.`***
+  * Send Graceful restart trigger from ATE:Port1
+  * Expected behavior is the same as RT-1.35.2.b
+  * Repeat by sending Graceful restart trigger from ATE:Port2 with the same expected behavior as RT-1.35.2.b<br><br><br>
+  
+**RT-1.35.5**
+
+  ***`a. DUT Helper for a restarting EBGP speaker whose BGP process was killed abruptly. In this case ERR policy is attached to the BGP neighborship.`***
+  * Start traffic. Send `gNOI.killProcessRequest_Signal_KILL` as per `gNOI proto` to ATE:Port1 to stop its BGP process abruptly. Configure ATE:Port1 to delay the BGP reestablishment for 330 secs over the Hold-time. Expected behavior in this case is the same as RT-1.35.2.a
+  * Post 330Secs over Hold-time expiry, BGP on ATE:Port1 is expected to be up and all traffic is expected to be successful.<br>
+  * Repeat the same test on ATE:Port2
+  
+  ***`b. DUT Helper for a restarting EBGP speaker whose BGP process was killed abruptly after ERR policy was removed.`***
+  * Start traffic. Send `gNOI.killProcessRequest_Signal_KILL` as per `gNOI proto` to ATE:Port1 to stop its BGP process abruptly. Configure ATE:Port1 to delay the BGP reestablishment for 330 secs over the Hold-time. Expected behavior in this case is the same as RT-1.35.2.b
+  * Post 330Secs over Hold-time expiry, BGP on ATE:Port1 is expected to be up and all traffic is expected to be successful.
+  * Repeat the same test on ATE:Port2<br><br><br>
+
+   
+
+ 
+**RT-1.35.6**
+  
+```TODO: gNOI.ClearBGPNeighborRequest_GRACEFUL used in this case is under review in https://github.com/openconfig/gnoi/pull/214```
+  
+  ***`a. Expected behavior when "Administrative Reset" Notification (rfc4486) sent to the peer while the ERR policy is attached to the neighborship.`***
+   *   Start traffic as per the flows above
+   *   Trigger BGP Notification (code 6 subocde 4) from DUT:Port1 towards ATE:Port1. Please use the `gNOI.ClearBGPNeighborRequest_GRACEFUL` message.
+   *   Cease notification of Code 6, subcode 4 will result in tcp connection reset but the routes aren't flushed
```

Comment:
Why is the "routes aren't flushed" applicable to the ATE session?

Reply:
Well, ERR is an extension to RFC8538 which is again an extension to RFC4724. While the last introduced GR, it excluded NOTIFICATION messages. Hence implementations supporting RFC4724 will flush routes for all kinds of NOTIFICATION messages. RFC8538 is an update to include NOTIFICATION messages as well part of GR. However, for RFC8538 it became necessary to introduce a new NOTIFICATION message called "HARD RESET" to indeed allow for flushing of routes voluntarily (more in https://datatracker.ietf.org/doc/html/rfc8538#section-5.1). Hence, not all messages cause Route flush. While "Administrative RESET" can cause TCP termination, voluntary route flush is possible w/ HARD_RESET (Code 6 subcode 9). 
Hope this explains and differentiates the case this Sub test case is testing from actual Route flush scenarios demonstrated in other sub tests.